Packs/Jira/Integrations/JiraV2/JiraV2.yml

```
@@ -7,109 +7,130 @@ configuration:
   name: url
   required: true
   type: 0
+  defaultvalue:
+  additionalinfo:
 - display: Username (Basic Authentication)
   name: username
   required: false
   type: 0
+  defaultvalue:
+  additionalinfo:
 - display: Password (Deprecated - Use API token)
   name: password
   required: false
   type: 4
+  defaultvalue:
+  additionalinfo:
 - displaypassword: API token (Recommended)
   name: credentials
   required: false
   type: 9
   hiddenusername: true
+  display:
+  defaultvalue:
+  additionalinfo:
 - defaultvalue: OauthKey
-  additionalinfo: Will be ignored if other required fields are not provided (for example both the Access Token and the Private Key).
+  additionalinfo: Will be ignored if other required fields are not provided (for example
+    both the Access Token and the Private Key).
   display: Consumer key (OAuth 1.0)
   name: consumerKey
   required: false
   type: 0
 - display: Access Token
-  additionalinfo: Used for both OAuth 1.0 and Personal Access Token authentication methods.
+  additionalinfo: Used for both OAuth 1.0 and Personal Access Token authentication
+    methods.
   name: accessToken
   required: false
   type: 0
+  defaultvalue:
 - display: Private key (PKCS8 format)
   name: privateKey
   required: false
   type: 14
+  defaultvalue:
+  additionalinfo:
 - defaultvalue: status!=done
   display: Query (in JQL) for fetching incidents
   name: query
   required: false
   type: 0
+  additionalinfo:
 - additionalinfo: This parameter is dismissed while "Use created field to fetch incidents"
     is checked.
   display: Issue index to start fetching incidents from
   name: idOffset
   required: false
   type: 0
+  defaultvalue:
 - display: Trust any certificate (not secure)
   name: insecure
   required: false
   type: 8
+  defaultvalue: ''
+  additionalinfo:
 - defaultvalue: 'false'
   display: Use system proxy settings
   name: proxy
   required: false
   type: 8
-- display: Fetch incidents
-  name: isFetch
-  required: false
-  type: 8
-- defaultvalue: 'false'
-  display: Mirror incoming incidents
-  hidden: false
+  additionalinfo:
+- display: Mirror incoming incidents
   name: incoming_mirror
   required: false
   type: 8
-- display: Incident type
-  name: incidentType
-  required: false
-  type: 13
-- display: Use created field to fetch incidents
+  defaultvalue: "false"
+  additionalinfo:
+- defaultvalue: "true"
```

Please revert - you can change to `false`.
```suggestion
- defaultvalue: "false"
```